libnopegl/src/backends/vk/gpu_ctx_vk.c

```diff
@@ -795,6 +795,12 @@ static int get_max_supported_samples(const VkPhysicalDeviceLimits *limits)
     return NGLI_MIN(max_color_samples, NGLI_MIN(max_depth_samples, max_stencil_samples));
 }
 
+static int get_max_color_attachments(const VkPhysicalDeviceLimits *limits)
+{
+    /* nope.gl only support up to 8 color attachments */
```

Comment:
I think the comment is unnecessary (even more so if you have a comment on `NGLI_MAX_COLOR_ATTACHMENTS` itself)

Reply:
Fixed.